spec/support/helpers/rack_helpers_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+RSpec.describe RackHelpers, type: :helper do
+  describe '#create_helper' do
+    subject(:helper) { create_rack_request(env) }
+
+    context 'without args' do
+      let(:env) { {} }
+
+      it 'returns Rack::Request instance double with default values' do
+        expect(helper).to be_an_instance_of(RSpec::Mocks::InstanceVerifyingDouble)
```

Comment:
Does this kind of expectations say something to us ?

Reply:
Yup. It's test for helper.